.circleci/config.yml

```diff
@@ -3,14 +3,17 @@ workflows:
   version: 2
   build-deploy:
     jobs:
-      - build:
+      - pr:
           filters:
             branches:
-              ignore: master
-      - master:
+              ignore: &integration_branches
+                - master
+                - develop
+                - /release\/\d+\.\d+
+      - integration:
           filters:
             branches:
-              only: master
+              only: *integration_branches
```

Comment:
How does CircleCI know what `integration_branches` are?

Reply:
That's a back reference to the array above on L9.

Reply:
That's a [YAML anchor reference](https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/) to the YAML anchor defined above with `&integration_branches`.